curl/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM ossfuzz/base-libfuzzer
+MAINTAINER dvyukov@google.com
+RUN apt-get install -y make autoconf automake libtool libssl-dev zlib1g-dev
+
```

Comment:
You will have `checkout_and_compile` working. It is only documented on base-libfuzzer page because the feature is not fully complete (no support for custom checkout scripts yet)